tools/network/MASTG-TOOL-0098.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Nope-Proxy
+platform: network
+source: https://github.com/summitt/Nope-Proxy
+---
+
+[Nope-Proxy](https://github.com/summitt/Nope-Proxy) is an extension for Burp that operates as a Non-HTTP Man-in-the-Middle (MiTM) tool, allow you to intercept both TCP and UDP traffic. Additionally, it allows DNS spoofing on mobile devices, enabling the rerouting of traffic to Burp for the interception of non-HTTP communications. Furthermore, it includes features such as manual traffic interception, automated traffic manipulation, a Python mangler, and traffic highlighting.
```

Comment:
MiTM is correct, but we're using MITM across the guide. Included also other small fixes:

```suggestion
[Nope-Proxy](https://github.com/summitt/Nope-Proxy) is an extension for Burp that operates as a Non-HTTP Man-in-the-Middle (MITM) tool, allowing you to intercept both TCP and UDP traffic. Additionally, it allows DNS spoofing on mobile devices to redirect traffic to Burp for the interception of non-HTTP communications. It also includes features such as manual traffic interception, automated traffic manipulation, a Python mangler, and traffic highlighting.
```